src/components/form/widgets/date-picker.ts

```diff
@@ -54,11 +54,16 @@ export class DatePicker extends React.Component {
         }
 
         const formatMapping = {
-            'date-time': 'YYYY-MM-DDTHH:mm:ss',
+            'date-time': 'YYYY-MM-DDTHH:mm:ssZ',
             date: 'YYYY-MM-DD',
             time: 'HH:mm:ss',
         };
-        const dateString = moment(event.detail).format(
+        const value =
+            props.schema.format !== 'date-time'
+                ? event.detail
+                : `${event.detail}Z`;
```

Comment:
I do not understand why this is needed, and it seems to mess up the time that you select. It will offset the time more than it should. Can we not just remove this?

Reply:
This also converts `event.detail` from an `object` to a `string` with `Z` at the end, that might be what causes the weird behavior

Reply:
Yea it should be removed.